microsoft-edge/extensions-chromium/publish/sidebar.md

```diff
@@ -0,0 +1,242 @@
+
+
+<!-- ------------------------------ -->
+#### Example 1: Default sidebar on every site 
+
+A sidebar can be set as default to show the same extension throughout all the tabs opened in the browser window. Default values will persist across sessions. 
+
+`manifest.json`:
+
+```json
+{
+  "name": "My side panel extension",
+  ...
+  "side_panel": {
+    "default_path": "sidepanel.html"
+  }
+  ...
+}
+```
+
+`sidepanel.html`:
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>My Sidepanel</title>
+  </head>
+  <body>
+    <h1>All sites sidepanel extension</h1>
+    <p>This side panel is enabled on all sites</p>
+  </body>
+</html>
+```
+
+
+<!-- ------------------------------ -->
+#### Example 2: Switching to a different sidebar
+
+Users can switch to a different sidebar when they switch tabs.
+
+The following example sets a welcome sidebar on `runtime.OnInstalled()`, when the user navigates to a different tab, it replaces the welcome page sidebar with the main sidebar.
+
+`service-worker.js`:
+
+```js
+const welcomePage = 'sidepanels/welcome-sp.html';
+const mainPage = 'sidepanels/main-sp.html';
+
+chrome.runtime.onInstalled.addListener(() => {
+  chrome.sidePanel.setOptions({ path: welcomePage });
+});
+
+chrome.tabs.onActivated.addListener(async ({ tabId }) => {
+  const { path } = await chrome.sidePanel.getOptions({ tabId });
+  if (path === welcomePage) {
+    chrome.sidePanel.setOptions({ path: mainPage });
+  }
+});
+```
+
+
+<!-- ------------------------------ -->
+#### Example 3: Enabling sidebar on a specific site
+
+An extension can use`sidepanel.setOptions()` to enable a sidebar on a specific site. This can be a particular website a user is using and would want the same extension to open in the sidebar, once this website is opened.
+
+`service-worker.js`:
+
+```js
+const GOOGLE_ORIGIN = 'https://www.google.com';
+
+chrome.tabs.onUpdated.addListener(async (tabId, info, tab) => {
+  if (!tab.url) return;
+  const url = new URL(tab.url);
+  // Enables the side panel on google.com
+  if (url.origin === GOOGLE_ORIGIN) {
+    await chrome.sidePanel.setOptions({
+      tabId,
+      path: 'sidepanel.html',
+      enabled: true
+    });
+  } else {
+    // Disables the side panel on all other sites
+    await chrome.sidePanel.setOptions({
+      tabId,
+      enabled: false
+    });
+  }
+});
+```
+
+When a user temporarily switches to a tab where the sidebar is not enabled, the sidebar will be hidden. 
+
+
+<!-- ------------------------------ -->
+#### Example 4: Enabling the action Icon to open the sidebar
+
+By declaring the “action” key in the manifest, developers can allow users to open the side panel when they click on the action toolbar icon with `sidePanel.setPanelBehavior()`.
+
+`manifest.json`:
+
+```json
+{
+  "name": "My side panel extension",
+  ...
+   "action": {
+    "default_title": "Click to open panel"
+  },
+  ...
+}
+```
+
+`service-worker.js`:
+
+```js
+const GOOGLE_ORIGIN = 'https://www.google.com';
```

Comment:
```suggestion
const GOOGLE_ORIGIN = 'https://www.bing.com';
```

Reply:
Outdated surrounding Markdown.
I incorp'd this change, per latest Files Changed tab
https://github.com/MicrosoftDocs/edge-developer/pull/2648/files